src/Compilers/CSharp/Portable/Symbols/AnonymousTypes/SynthesizedSymbols/AnonymousType.PropertySymbol.cs

```diff
@@ -19,7 +19,7 @@ internal sealed partial class AnonymousTypeManager
         /// <summary>
         /// Represents an anonymous type template's property symbol.
         /// </summary>
-        private sealed class AnonymousTypePropertySymbol : PropertySymbol
+        internal sealed class AnonymousTypePropertySymbol : PropertySymbol
```

Comment:
Is it necessary to make AnonymousTypePropertySymbol, AnonymousTypeTemplateSymbol, and NameAndIndex internal?